.github/workflows/cli.yml

```diff
@@ -73,265 +45,135 @@ jobs:
-        uses: actions/cache@v3
-        timeout-minutes: 5
-        continue-on-error: true
-        with:
-          path: ~/.cargo/*
-          key: ${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('Cargo.lock', 'Cargo.toml') }}
-
-      - name: set up a cache for target
-        uses: actions/cache@v3
-        timeout-minutes: 5
-        continue-on-error: true
-        with:
-          path: target
-          key: ${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('Cargo.lock', 'Cargo.toml') }}
-
-      - name: fetch assets
-        run: |
-          curl https://temp-artifact-storage.s3.eu-west-3.amazonaws.com/assets.zip --output assets.zip
-          unzip assets.zip
-          rm -r crates/server/assets
-          mv assets crates/server/assets
-          cp -r crates/server/assets ~/.grafbase/
-          touch ~/.grafbase/version.txt
-
-      - name: test
+      - name: Test
         uses: actions-rs/cargo@v1
         with:
           command: test
 
-      - name: build
+      - name: Build aarch64 release
         uses: actions-rs/cargo@v1
-        if: github.ref == 'refs/heads/main'
+        if: startsWith(github.ref, 'refs/tags/')
         with:
           use-cross: true
           command: build
           args: "--release --target aarch64-apple-darwin"
 
-      - name: build
+      - name: Build x86_64 release
         uses: actions-rs/cargo@v1
-        if: github.ref == 'refs/heads/main'
+        if: startsWith(github.ref, 'refs/tags/')
         with:
           use-cross: true
           command: build
           args: "--release --target x86_64-apple-darwin"
 
-      - name: chmod binary
-        if: github.ref == 'refs/heads/main'
-        run: chmod +x target/x86_64-apple-darwin/release/grafbase
-
-      - name: chmod binary
-        if: github.ref == 'refs/heads/main'
-        run: chmod +x target/aarch64-apple-darwin/release/grafbase
+      - name: chmod binaries
+        if: startsWith(github.ref, 'refs/tags/')
+        shell: bash
+        run: |
+          chmod +x target/x86_64-apple-darwin/release/grafbase
+          chmod +x target/aarch64-apple-darwin/release/grafbase
 
-      - name: archive production artifacts
+      - name: Upload aarch binary
+        if: startsWith(github.ref, 'refs/tags/')
         uses: actions/upload-artifact@v3
-        if: github.ref == 'refs/heads/main'
         with:
-          name: grafbase-x86_64-apple-darwin
-          path: target/x86_64-apple-darwin/release/grafbase
+          name: ${{ github.ref_name }}-aarch64-darwin
+          path: target/aarch64-apple-darwin/release/grafbase
 
-      - name: archive production artifacts
+      - name: Upload x86_64 binary
+        if: startsWith(github.ref, 'refs/tags/')
         uses: actions/upload-artifact@v3
-        if: github.ref == 'refs/heads/main'
         with:
-          name: grafbase-aarch64-apple-darwin
-          path: target/aarch64-apple-darwin/release/grafbase
+          name: ${{ github.ref_name }}-x86_64-darwin
```

Comment:
Why not use
`${{ github.ref_name }}-x86_64-apple-darwin`?

Reply:
sure I can add apple

Reply:
I think it's the same on the x86_64, not sure if other platforms have this